vcl_templates/www.vcl.erb

```diff
@@ -296,6 +296,9 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-Example=" req.http.GOVUK-ABTest-Example "; expires=" now + 1d;
   }
 
+  # Set the tls version cookie with the raw protocol version from fastly
+  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; max-age=0; secure";
```

Comment:
Rebased to include @timblair's idea of having a check to see if it already exists and not setting max-age or expires.